go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/containerd/typeurl/v2 v2.1.1
 	github.com/creack/pty v1.1.18
 	github.com/distribution/reference v0.5.0
-	github.com/docker/cli v26.0.0+incompatible
+	github.com/docker/cli v26.0.1-0.20240401150816-155dc5e4e406+incompatible
```

Comment:
Do we want the change backported to the 26.0  branch so that we don't have to force consumers to depend on master? (Haven't checked if we can)

cc @vvoland @laurazard